### PR TITLE
java 13-17 new features

### PR DESCRIPTION
**Pull request by esacaceres**
Java 13 & 14

### CODE REVIEW

File: _posts/2021-02-19-java-new-features-by-version.md [hunk 1]
```diff
@@ -487,4 +528,149 @@ public class mybenchmark {
...
+    "</html>\n";
+
+//In Java 13 also can be defined as:
+String html = """
+  <html>
+      <body>
+          <p class="text">Hello, Text Blocks</p>
+      </body>
+  </html>""";
+</code></pre>
```
Comment (jdeflaux) [inline _posts/2021-02-19-java-new-features-by-version.md:550]: I was looking to do this last week in camunda 🍾

File: _posts/2021-02-19-java-new-features-by-version.md [hunk 2]
```diff
@@ -487,4 +528,149 @@ public class mybenchmark {
...
+        PhoneNumber that = (PhoneNumber) o;
+        return super.equals(that)
+            && Objects.equals(this.lineNumber, that.lineNumber)
+            && Objects.equals(this.prefix, that.prefix)
+            && Objects.equals(this.areaCode, that.areaCode);
+    }
+}
+
+// Now AWESOME simplification!
+public record PhoneNumber(Integer lineNumber, Integer prefix, Integer areaCode) {
```
Comment (jdeflaux) [inline _posts/2021-02-19-java-new-features-by-version.md:630]: I like that

**Review by jdeflaux — COMMENTED** [review]
Seems good to me

please ask Alison and/or Simon to read it as well as it is published on our public website

**Review by jaortizbaz — APPROVED** [review]

**Comment by jaortizbaz** [comment]
Looks good!

**Review by chookie — APPROVED** [review]

**Pull request merged**